package.json

```diff
@@ -62,16 +62,16 @@
     "clipboard-polyfill": "^3.0.3",
     "dirty-chai": "^2.0.1",
     "eslint": "^8.3.0",
-    "jsdom": "^16.4.0",
-    "mocha": "^8.2.1",
+    "jsdom": "16.4.0",
```

Comment:
JSDOM has some changes between `16.4.0` and `16.5.0` that causes a couple tests to break, and I'm not sure what they are or how to fix them 😞 

<details>

<summary>
Test Log
</summary>

```
  2 failing

  1) document-capture/hooks/use-async
       returns suspense resource that renders error fallback:
     Uncaught Error: Caught falsy/undefined exception which would otherwise be uncaught. No stack trace found; try a debugger
      at process._fatalException (internal/process/execution.js:167:25)

  2) "after each" hook for "returns suspense resource that renders error fallback":
     Unexpected console logging: Error: Uncaught undefined
    at reportException (/Users/mitchellehenke/projects/identity-idp/node_modules/jsdom/lib/jsdom/living/helpers/runtime-script-errors.js:66:24)
    at innerInvokeEventListeners (/Users/mitchellehenke/projects/identity-idp/node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:341:9)
    at invokeEventListeners (/Users/mitchellehenke/projects/identity-idp/node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:274:3)
    at HTMLUnknownElementImpl._dispatch (/Users/mitchellehenke/projects/identity-idp/node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:221:9)
    at HTMLUnknownElementImpl.dispatchEvent (/Users/mitchellehenke/projects/identity-idp/node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:94:17)
    at HTMLUnknownElement.dispatchEvent (/Users/mitchellehenke/projects/identity-idp/node_modules/jsdom/lib/jsdom/living/generated/EventTarget.js:231:34)
    at Object.invokeGuardedCallbackDev (/Users/mitchellehenke/projects/identity-idp/node_modules/react-dom/cjs/react-dom.development.js:3994:16)
    at invokeGuardedCallback (/Users/mitchellehenke/projects/identity-idp/node_modules/react-dom/cjs/react-dom.development.js:4056:31)
    at beginWork$1 (/Users/mitchellehenke/projects/identity-idp/node_modules/react-dom/cjs/react-dom.development.js:23964:7)
    at performUnitOfWork (/Users/mitchellehenke/projects/identity-idp/node_modules/react-dom/cjs/react-dom.development.js:22779:12)
    at workLoopSync (/Users/mitchellehenke/projects/identity-idp/node_modules/react-dom/cjs/react-dom.development.js:22707:5)
    at renderRootSync (/Users/mitchellehenke/projects/identity-idp/node_modules/react-dom/cjs/react-dom.development.js:22670:7)
    at performSyncWorkOnRoot (/Users/mitchellehenke/projects/identity-idp/node_modules/react-dom/cjs/react-dom.development.js:22293:18)
    at /Users/mitchellehenke/projects/identity-idp/node_modules/react-dom/cjs/react-dom.development.js:11327:26
    at unstable_runWithPriority (/Users/mitchellehenke/projects/identity-idp/node_modules/scheduler/cjs/scheduler.development.js:468:12)
    at runWithPriority$1 (/Users/mitchellehenke/projects/identity-idp/node_modules/react-dom/cjs/react-dom.development.js:11276:10)
    at flushSyncCallbackQueueImpl (/Users/mitchellehenke/projects/identity-idp/node_modules/react-dom/cjs/react-dom.development.js:11322:9)
    at workLoop (/Users/mitchellehenke/projects/identity-idp/node_modules/scheduler/cjs/scheduler.development.js:417:34)
    at flushWork (/Users/mitchellehenke/projects/identity-idp/node_modules/scheduler/cjs/scheduler.development.js:390:14)
    at Timeout._flushCallback [as _onTimeout] (/Users/mitchellehenke/projects/identity-idp/node_modules/scheduler/cjs/scheduler.development.js:55:9)
    at listOnTimeout (internal/timers.js:557:17)
    at processTimers (internal/timers.js:500:7) undefined, The above error occurred in the <Child> component:

    at Child (/Users/mitchellehenke/projects/identity-idp/spec/javascripts/packages/document-capture/hooks/use-async-spec.jsx:19:5)
    at Suspense
    at SuspenseErrorBoundary (/Users/mitchellehenke/projects/identity-idp/app/javascript/packages/document-capture/components/suspense-error-boundary.jsx:31:5)
    at Parent (/Users/mitchellehenke/projects/identity-idp/spec/javascripts/packages/document-capture/hooks/use-async-spec.jsx:26:5)
    at UploadContextProvider (/Users/mitchellehenke/projects/identity-idp/app/javascript/packages/document-capture/context/upload.jsx:116:3)
    at wrapper (/Users/mitchellehenke/projects/identity-idp/spec/javascripts/support/document-capture.jsx:60:7)

React will try to recreate this component tree from scratch using the error boundary you provided, SuspenseErrorBoundary., Error: Uncaught undefined
    at reportException (/Users/mitchellehenke/projects/identity-idp/node_modules/jsdom/lib/jsdom/living/helpers/runtime-script-errors.js:66:24)
    at innerInvokeEventListeners (/Users/mitchellehenke/projects/identity-idp/node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:341:9)
    at invokeEventListeners (/Users/mitchellehenke/projects/identity-idp/node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:274:3)
    at HTMLUnknownElementImpl._dispatch (/Users/mitchellehenke/projects/identity-idp/node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:221:9)
    at HTMLUnknownElementImpl.dispatchEvent (/Users/mitchellehenke/projects/identity-idp/node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:94:17)
    at HTMLUnknownElement.dispatchEvent (/Users/mitchellehenke/projects/identity-idp/node_modules/jsdom/lib/jsdom/living/generated/EventTarget.js:231:34)
    at Object.invokeGuardedCallbackDev (/Users/mitchellehenke/projects/identity-idp/node_modules/react-dom/cjs/react-dom.development.js:3994:16)
    at invokeGuardedCallback (/Users/mitchellehenke/projects/identity-idp/node_modules/react-dom/cjs/react-dom.development.js:4056:31)
    at beginWork$1 (/Users/mitchellehenke/projects/identity-idp/node_modules/react-dom/cjs/react-dom.development.js:23964:7)
    at performUnitOfWork (/Users/mitchellehenke/projects/identity-idp/node_modules/react-dom/cjs/react-dom.development.js:22779:12)
    at workLoopSync (/Users/mitchellehenke/projects/identity-idp/node_modules/react-dom/cjs/react-dom.development.js:22707:5)
    at renderRootSync (/Users/mitchellehenke/projects/identity-idp/node_modules/react-dom/cjs/react-dom.development.js:22670:7)
    at performSyncWorkOnRoot (/Users/mitchellehenke/projects/identity-idp/node_modules/react-dom/cjs/react-dom.development.js:22293:18)
    at /Users/mitchellehenke/projects/identity-idp/node_modules/react-dom/cjs/react-dom.development.js:11327:26
    at unstable_runWithPriority (/Users/mitchellehenke/projects/identity-idp/node_modules/scheduler/cjs/scheduler.development.js:468:12)
    at runWithPriority$1 (/Users/mitchellehenke/projects/identity-idp/node_modules/react-dom/cjs/react-dom.development.js:11276:10)
    at flushSyncCallbackQueueImpl (/Users/mitchellehenke/projects/identity-idp/node_modules/react-dom/cjs/react-dom.development.js:11322:9)
    at workLoop (/Users/mitchellehenke/projects/identity-idp/node_modules/scheduler/cjs/scheduler.development.js:417:34)
    at flushWork (/Users/mitchellehenke/projects/identity-idp/node_modules/scheduler/cjs/scheduler.development.js:390:14)
    at Timeout._flushCallback [as _onTimeout] (/Users/mitchellehenke/projects/identity-idp/node_modules/scheduler/cjs/scheduler.development.js:55:9)
    at listOnTimeout (internal/timers.js:557:17)
    at processTimers (internal/timers.js:500:7) undefined, The above error occurred in the <Child> component:

    at Child (/Users/mitchellehenke/projects/identity-idp/spec/javascripts/packages/document-capture/hooks/use-async-spec.jsx:19:5)
    at Suspense
    at SuspenseErrorBoundary (/Users/mitchellehenke/projects/identity-idp/app/javascript/packages/document-capture/components/suspense-error-boundary.jsx:31:5)
    at Parent (/Users/mitchellehenke/projects/identity-idp/spec/javascripts/packages/document-capture/hooks/use-async-spec.jsx:26:5)
    at UploadContextProvider (/Users/mitchellehenke/projects/identity-idp/app/javascript/packages/document-capture/context/upload.jsx:116:3)
    at wrapper (/Users/mitchellehenke/projects/identity-idp/spec/javascripts/support/document-capture.jsx:60:7)

Consider adding an error boundary to your tree to customize error handling behavior.
Visit https://reactjs.org/link/error-boundaries to learn more about error boundaries.: expected [ Array(4) ] to be empty
  AssertionError: Unexpected console logging: Error: Uncaught undefined
      at reportException (node_modules/jsdom/lib/jsdom/living/helpers/runtime-script-errors.js:66:24)
      at innerInvokeEventListeners (node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:341:9)
      at invokeEventListeners (node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:274:3)
      at HTMLUnknownElementImpl._dispatch (node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:221:9)
      at HTMLUnknownElementImpl.dispatchEvent (node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:94:17)
      at HTMLUnknownElement.dispatchEvent (node_modules/jsdom/lib/jsdom/living/generated/EventTarget.js:231:34)
      at Object.invokeGuardedCallbackDev (node_modules/react-dom/cjs/react-dom.development.js:3994:16)
      at invokeGuardedCallback (node_modules/react-dom/cjs/react-dom.development.js:4056:31)
      at beginWork$1 (node_modules/react-dom/cjs/react-dom.development.js:23964:7)
      at performUnitOfWork (node_modules/react-dom/cjs/react-dom.development.js:22779:12)
      at workLoopSync (node_modules/react-dom/cjs/react-dom.development.js:22707:5)
      at renderRootSync (node_modules/react-dom/cjs/react-dom.development.js:22670:7)
      at performSyncWorkOnRoot (node_modules/react-dom/cjs/react-dom.development.js:22293:18)
      at /Users/mitchellehenke/projects/identity-idp/node_modules/react-dom/cjs/react-dom.development.js:11327:26
      at unstable_runWithPriority (node_modules/scheduler/cjs/scheduler.development.js:468:12)
      at runWithPriority$1 (node_modules/react-dom/cjs/react-dom.development.js:11276:10)
      at flushSyncCallbackQueueImpl (node_modules/react-dom/cjs/react-dom.development.js:11322:9)
      at workLoop (node_modules/scheduler/cjs/scheduler.development.js:417:34)
      at flushWork (node_modules/scheduler/cjs/scheduler.development.js:390:14)
      at Timeout._flushCallback [as _onTimeout] (node_modules/scheduler/cjs/scheduler.development.js:55:9)
      at listOnTimeout (internal/timers.js:557:17)
      at processTimers (internal/timers.js:500:7) undefined, The above error occurred in the <Child> component:

      at Child (spec/javascripts/packages/document-capture/hooks/use-async-spec.jsx:19:5)
      at Suspense
      at SuspenseErrorBoundary (app/javascript/packages/document-capture/components/suspense-error-boundary.jsx:31:5)
      at Parent (spec/javascripts/packages/document-capture/hooks/use-async-spec.jsx:26:5)
      at UploadContextProvider (app/javascript/packages/document-capture/context/upload.jsx:116:3)
      at wrapper (spec/javascripts/support/document-capture.jsx:60:7)

  React will try to recreate this component tree from scratch using the error boundary you provided, SuspenseErrorBoundary., Error: Uncaught undefined
      at reportException (node_modules/jsdom/lib/jsdom/living/helpers/runtime-script-errors.js:66:24)
      at innerInvokeEventListeners (node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:341:9)
      at invokeEventListeners (node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:274:3)
      at HTMLUnknownElementImpl._dispatch (node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:221:9)
      at HTMLUnknownElementImpl.dispatchEvent (node_modules/jsdom/lib/jsdom/living/events/EventTarget-impl.js:94:17)
      at HTMLUnknownElement.dispatchEvent (node_modules/jsdom/lib/jsdom/living/generated/EventTarget.js:231:34)
      at Object.invokeGuardedCallbackDev (node_modules/react-dom/cjs/react-dom.development.js:3994:16)
      at invokeGuardedCallback (node_modules/react-dom/cjs/react-dom.development.js:4056:31)
      at beginWork$1 (node_modules/react-dom/cjs/react-dom.development.js:23964:7)
      at performUnitOfWork (node_modules/react-dom/cjs/react-dom.development.js:22779:12)
      at workLoopSync (node_modules/react-dom/cjs/react-dom.development.js:22707:5)
      at renderRootSync (node_modules/react-dom/cjs/react-dom.development.js:22670:7)
      at performSyncWorkOnRoot (node_modules/react-dom/cjs/react-dom.development.js:22293:18)
      at /Users/mitchellehenke/projects/identity-idp/node_modules/react-dom/cjs/react-dom.development.js:11327:26
      at unstable_runWithPriority (node_modules/scheduler/cjs/scheduler.development.js:468:12)
      at runWithPriority$1 (node_modules/react-dom/cjs/react-dom.development.js:11276:10)
      at flushSyncCallbackQueueImpl (node_modules/react-dom/cjs/react-dom.development.js:11322:9)
      at workLoop (node_modules/scheduler/cjs/scheduler.development.js:417:34)
      at flushWork (node_modules/scheduler/cjs/scheduler.development.js:390:14)
      at Timeout._flushCallback [as _onTimeout] (node_modules/scheduler/cjs/scheduler.development.js:55:9)
      at listOnTimeout (internal/timers.js:557:17)
      at processTimers (internal/timers.js:500:7) undefined, The above error occurred in the <Child> component:

      at Child (spec/javascripts/packages/document-capture/hooks/use-async-spec.jsx:19:5)
      at Suspense
      at SuspenseErrorBoundary (app/javascript/packages/document-capture/components/suspense-error-boundary.jsx:31:5)
      at Parent (spec/javascripts/packages/document-capture/hooks/use-async-spec.jsx:26:5)
      at UploadContextProvider (app/javascript/packages/document-capture/context/upload.jsx:116:3)
      at wrapper (spec/javascripts/support/document-capture.jsx:60:7)

  Consider adding an error boundary to your tree to customize error handling behavior.
  Visit https://reactjs.org/link/error-boundaries to learn more about error boundaries.: expected [ Array(4) ] to be empty
      at Context.<anonymous> (spec/javascripts/support/console.js:58:43)
      at processImmediate (internal/timers.js:464:21)



/Users/mitchellehenke/projects/identity-idp/node_modules/mocha/lib/runner.js:962
    throw err;
    ^

Error: done() called multiple times in test <document-capture/hooks/use-async returns suspense resource that renders error fallback> of file /Users/mitchellehenke/projects/identity-idp/spec/javascripts/packages/document-capture/hooks/use-async-spec.jsx; in addition, done() received error: Error: Unable to find an element with the text: Error. This could be because the text is broken up by multiple elements. In this case, you can provide a function for your text matcher to make your matcher more flexible.

<body
  class=""
>
  <div />
</body>
    at waitForWrapper (/Users/mitchellehenke/projects/identity-idp/node_modules/@testing-library/dom/dist/wait-for.js:166:27)
    at findByText (/Users/mitchellehenke/projects/identity-idp/node_modules/@testing-library/dom/dist/query-helpers.js:100:77)
    at Context.<anonymous> (/Users/mitchellehenke/projects/identity-idp/spec/javascripts/packages/document-capture/hooks/use-async-spec.jsx:70:18)
    at callFn (/Users/mitchellehenke/projects/identity-idp/node_modules/mocha/lib/runnable.js:366:21)
    at Test.Runnable.run (/Users/mitchellehenke/projects/identity-idp/node_modules/mocha/lib/runnable.js:354:5)
    at Runner.runTest (/Users/mitchellehenke/projects/identity-idp/node_modules/mocha/lib/runner.js:678:10)
    at /Users/mitchellehenke/projects/identity-idp/node_modules/mocha/lib/runner.js:801:12
    at next (/Users/mitchellehenke/projects/identity-idp/node_modules/mocha/lib/runner.js:593:14)
    at /Users/mitchellehenke/projects/identity-idp/node_modules/mocha/lib/runner.js:603:7
    at next (/Users/mitchellehenke/projects/identity-idp/node_modules/mocha/lib/runner.js:486:14)
    at Immediate.<anonymous> (/Users/mitchellehenke/projects/identity-idp/node_modules/mocha/lib/runner.js:571:5)
    at processImmediate (internal/timers.js:464:21) {
  name: 'TestingLibraryElementError'
}
    at createMultipleDoneError (/Users/mitchellehenke/projects/identity-idp/node_modules/mocha/lib/errors.js:428:13)
    at multiple (/Users/mitchellehenke/projects/identity-idp/node_modules/mocha/lib/runnable.js:290:24)
    at done (/Users/mitchellehenke/projects/identity-idp/node_modules/mocha/lib/runnable.js:301:14)
    at /Users/mitchellehenke/projects/identity-idp/node_modules/mocha/lib/runnable.js:377:11 {
  code: 'ERR_MOCHA_MULTIPLE_DONE',
  valueType: 'object',
  value: Error: Unable to find an element with the text: Error. This could be because the text is broken up by multiple elements. In this case, you can provide a function for your text matcher to make your matcher more flexible.

  <body
    class=""
  >
    <div />
  </body>
      at waitForWrapper (/Users/mitchellehenke/projects/identity-idp/node_modules/@testing-library/dom/dist/wait-for.js:166:27)
      at findByText (/Users/mitchellehenke/projects/identity-idp/node_modules/@testing-library/dom/dist/query-helpers.js:100:77)
      at Context.<anonymous> (/Users/mitchellehenke/projects/identity-idp/spec/javascripts/packages/document-capture/hooks/use-async-spec.jsx:70:18)
      at callFn (/Users/mitchellehenke/projects/identity-idp/node_modules/mocha/lib/runnable.js:366:21)
      at Test.Runnable.run (/Users/mitchellehenke/projects/identity-idp/node_modules/mocha/lib/runnable.js:354:5)
      at Runner.runTest (/Users/mitchellehenke/projects/identity-idp/node_modules/mocha/lib/runner.js:678:10)
      at /Users/mitchellehenke/projects/identity-idp/node_modules/mocha/lib/runner.js:801:12
      at next (/Users/mitchellehenke/projects/identity-idp/node_modules/mocha/lib/runner.js:593:14)
      at /Users/mitchellehenke/projects/identity-idp/node_modules/mocha/lib/runner.js:603:7
      at next (/Users/mitchellehenke/projects/identity-idp/node_modules/mocha/lib/runner.js:486:14)
      at Immediate.<anonymous> (/Users/mitchellehenke/projects/identity-idp/node_modules/mocha/lib/runner.js:571:5)
      at processImmediate (internal/timers.js:464:21) {
    name: 'TestingLibraryElementError'
  }
}
error Command failed with exit code 7.
```

</details>